app/models/local_authority_district.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class LocalAuthorityDistrict < ApplicationRecord
+  has_many :schools
```

Comment:
The same as in other pull request - if we delete local authority district - do we expect schools to be deleted as well or do we want them as orphaned records.

Reply:
In this instance I think they should be kept as orphaned

Reply:
Yeah, deleting schools is a bit scary - especially since any LA / LAD deletions will cone from GIAS, which, uhmm, has a reputation for being questionably consistent at times.

Reply:
Cool cool